src/sentry/testutils/cases.py

```diff
@@ -1128,3 +1130,43 @@ def create_user_member_role(self):
             user=self.user, organization=self.organization, role="member", teams=[self.team]
         )
         self.login_as(self.user)
+
+
+class TestMigrations(TestCase):
+    """
+    From https://www.caktusgroup.com/blog/2016/02/02/writing-unit-tests-django-migrations/
+    """
+
+    @property
+    def app(self):
+        return "sentry"
+        return apps.get_containing_app_config(type(self).__module__).name
```

Comment:
@wedamija did you mean to leave this second return statement?

Reply:
Oops, no, I'll remove. Thanks!